### PR TITLE
Feat: Add settings modal translations for all remaining languages

### PR DESCRIPTION
**Pull request by OndraM**

### CODE REVIEW

File: src/languages/de.ts [hunk 1]
```diff
@@ -33,53 +33,57 @@ export const config = (extraMessages: ExtraMessages): VanillaCookieConsent.Langu
...
           toggle: {
             value: CookieConsentCategory.FUNCTIONALITY,
             enabled: false,
             readonly: false,
           },
         },
         {
-          title: 'AD TODO',
-          description: `Lorem ipsum`,
+          title: 'Marketing-Cookies',
```
Comment (literat) [inline src/languages/de.ts:76]: `Marketing-Cookies` -> `Marketing Cookies` ?

**Comment by literat** [comment]
I have a problem with some wording because it sounds weird to me. Like in English `your own settings`. Or Czech or Slovak `vlastní nastavení`. There is nothing mine on a webpage unless I have access to some kind of an admin panel. So maybe `in the settings` is better, or `in the custom settings`.

We can use `Nastavení cookies` in Czech instead of `Vlastní nastavení` because I think it is not a correct translation of a `custom settings`, maybe `podrobné nastavení`. Maybe the same in other languages.

What do you think?

**Review by literat — CHANGES_REQUESTED** [review]

**Comment by OndraM** [comment]
> What do you think?

This would have to be discussed elsewhere - this is just an implementation of CCM-55 .

File: src/languages/de.ts (hunk 1 above)
Comment (OndraM) [reply src/languages/de.ts:76]: Updated

**Comment by OndraM** [comment]
@literat Translations updated.

**Review by literat — APPROVED** [review]

**Pull request merged**